plugins/modules/cloud/misc/terraform.py

```diff
@@ -93,6 +93,12 @@
     type: list
     elements: path
     version_added: '0.2.0'
+  check_destroy:
+    description:
+      - Check and apply only when there are no deletes to resources.
```

Comment:
```suggestion
      - Check and apply only when there are no deletes to resources. This option will be ignored if state=absent.
```

Reply:
Sure,  i forgot to add that 😁

Reply:
I updated my suggestion to make the option behavior more clear.